tools/linter/components/Operation.ts

```diff
@@ -60,6 +60,8 @@ export default class Operation extends ValidatorBase {
         const description = this.spec.description;
         if(!description || description === '')
             return this.error(`Missing description property`);
+        if(!description.endsWith('.'))
```

Comment:
Speaking of periods, standardize errors (in another PR is fine). Above you have one that looks like a full sentence followed by a non-full sentence. I think it's easier to make everything be a full sentence.

```
Invalid x-operation-group '${this.group}'. The '_core' namespace must be omitted in x-operation-group.
```

```
Missing description property.
```